src/reducers/FormLogin/index.ts

```diff
@@ -0,0 +1,24 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+import { formLoginReducers, extraReducers } from './actions';
+
+export interface FormLoginState {
```

Comment:
We might re-use this interface again, should we re-name it to `FormState`?

Reply:
Changed by moving into `type` folder here https://github.com/Lahphim/react-surveyor-challenge/pull/65/commits/66a21d34220d8e1268c7efb0c44bdd1fba72e5f1